app/actors/file_set_attach_files_actor.rb

```diff
@@ -0,0 +1,33 @@
+# Actions are decoupled from controller logic so that they may be called from a controller or a background job.
+class FileSetAttachFilesActor < Hyrax::Actors::FileSetActor
```

Comment:
What change was needed here that required us overriding the (presumably) upstream `FileSetAttachFilesActor`?